src/cpu/platform.hpp

```diff
@@ -30,11 +30,12 @@
 // - DNNL_PPC64
 // - DNNL_S390X
 // - DNNL_RV64
+// - DNNL_LOONGARCH64
 // - DNNL_ARCH_GENERIC
 // Target architecture macro is set to 1, others to 0. All macros are defined.
 
 #if defined(DNNL_X64) + defined(DNNL_AARCH64) + defined(DNNL_PPC64) \
-                + defined(DNNL_S390X) + defined(DNNL_RV64) \
+                + defined(DNNL_S390X) + defined(DNNL_RV64) + defined(DNNL_LOONGARCH64) \
```

Comment:
Please apply clang-format on this file.

Reply:
> Please apply clang-format on this file.

It's been revised and resubmitted, thanks